README.md

```diff
@@ -104,6 +110,25 @@ async function runSample() {
 runSample().catch(console.error);
 ```
 
+You can also make calls directly to the APIs by installing a submodule:
+
+``` js
+const docs = require('@googleapis/docs')
+
+const client = await docs.docs({
+    version: 'v1',
+    auth: 'YOUR_AUTHENTICATED_CLIENT'
```

Comment:
Assume people will just copy and paste these samples.  When you use a string for `auth`, it assumes the service accepts an API key, which most of them don't do.  Instead - can we show something closer to what a user would do for a simple example?